src/components/VisualList/index.js

```diff
@@ -0,0 +1,143 @@
+import * as React from "react";
+import { Container, Box, Flex, Grid, GridItem, Text, Heading, useColorModeValue as mode } from "@chakra-ui/react";
+import { renderRichText } from "gatsby-source-contentful/rich-text";
+import PhotoWrapper from "SVG/PhotoWrapper";
+import { BLOCKS } from "@contentful/rich-text-types";
+
+const VisualList = ({ id, title, variant, cards }) => {
+  const TextWrapper = ({ children }) => (
+    <Text color={`#00ADBC`}>
```

Comment:
@rudymacias  this color can be changed to `primary` because its one of the base colors we set up for chakra. You can add more colors in the `@chakra-ui/gatsby-plugin/theme/foundations/colors.js` file